src/core/style/layout.scss

```diff
@@ -135,7 +135,23 @@
       box-sizing: border-box;
     }
 
-    .layout-row > .#{$flexName}-#{$i * 5},
+    .layout-row > .#{$flexName}-#{$i * 5} {
```

Comment:
This will not work in you have markup like this:

```html
<div layout="row" layout-sm="column">
  <div flex="33"  flex-sm="95">
  <div flex="33"  flex-sm="95">
  <div flex="33"  flex-sm="95">
</div>
```